index.html

```diff
@@ -1,197 +1,274 @@
+  <link rel="stylesheet" href="style.css" />
 
-  <body>
-    <div class="wrapper">
-      <header class="head-toolbar">
-        <a class="toolbar-title" href="#"> Welcome </a>
-        <nav class="navbar">
-          <a href="#"> <img src="assets/images/hamburger.svg" alt="" /> </a>
-        </nav>
-      </header>
+  <title>Yousef Hesham</title>
+</head>
 
-      <section class="headline">
-        <h2 class="welcome-message">
-          Hey there.I’m Yousef <br />
-          <span>I’m a software developer </span>
-        </h2>
-        <p class="description">
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
-        <div class="social">
-          <ul class="social-icons">
-            <li class="items">
-              <a href="#"> <img src="assets/images/twitter.svg" alt="" /></a>
+<body>
+  <div class="wrapper">
+
+    <header class="head-toolbar">
+      <a class="toolbar-title mobile-version"  href="#"> Welcome </a>
+      <nav class="navbar">
+        <a href="#" class="mobile-version"> <img src="assets/images/hamburger.svg" alt="" /> </a>
+        <ul class="nav-list">
+          <li class="nav-item"><a href="#works"> Portfolio</a></li>
+          <li class="nav-item"><a href="#about"> About</a></li>
+          <li class="nav-item"><a href="#contact"> Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="headline">
+      <h2 class="welcome-message">
+        Hey there.I’m Yousef <br />
+        <span>I’m a software developer </span>
+      </h2>
+      <p class="description">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded,<br> don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <ul class="social-icons">
+          <li class="items">
+            <a href="#"> <img src="assets/images/twitter.svg" alt="" /></a>
+          </li>
+          <li class="items">
+            <a href="#">
+              <img src="assets/images/Linkedinicon.svg" alt="" />
+            </a>
+          </li>
+          <li class="items">
+            <a href="#"> <img src="assets/images/medium.svg" alt="" /> </a>
+          </li>
+          <li class="items">
+            <a href="#"> <img src="assets/images/github.svg" alt="" /> </a>
+          </li>
+          <li class="items">
+            <a href="#"> <img src="assets/images/angellist.svg" alt="" /> </a>
+          </li>
+        </ul>
+
+      </div>
+      <img class="disabled-button" src="assets/images/Disabled.svg" alt="" />
+
+    </section>
+   </div> 
+
+    <section class="recent-works" id="works">
+      <h2 class="section-title">My Recent Works</h2>
+      <hr class="horizontal-line" />
```

Comment:
- [ ] OPTIONAL: I recommend adding background images to make your page look even better: 
- [ ] Please add more space between about me and recent works section.

Your design  | Figma design
------------- | -------------
<img width="866" alt="Screenshot 2022-03-24 at 12 26 57" src="https://user-images.githubusercontent.com/77038610/159874019-1c0bef72-647f-41bc-a284-fe66f6b437c8.png"> | <img width="712" alt="Screenshot 2022-03-24 at 12 27 19" src="https://user-images.githubusercontent.com/77038610/159874086-c1f741ac-0a4c-4fcf-9b83-d934216bff2c.png">